### ISSUE TITLE
Still cannot create NTRU Keys in 0.4.4 (Trac #1929)

### ISSUE THREAD

**Issue by str4d** [i2p/i2p.i2p-bote#88]
You marked the issue as fixed but the error is still there. Is there an update to the fixed version? I can't find any new updates to it.

Migrated from https://trac.i2p2.de/ticket/1929
```json
{
    "status": "closed", 
    "changetime": "2017-01-20T16:15:45", 
    "description": "You marked the issue as fixed but the error is still there. Is there an update to the fixed version? I can't find any new updates to it.", 
    "reporter": "WAR10CKumbra", 
    "cc": "", 
    "resolution": "duplicate", 
    "_ts": "1484928945615480", 
    "component": "apps/plugins", 
    "summary": "Still cannot create NTRU Keys in 0.4.4", 
    "priority": "minor", 
    "keywords": "I2P-Bote", 
    "version": "0.9.28", 
    "parents": "", 
    "time": "2017-01-15T00:55:21", 
    "milestone": "0.9.29", 
    "owner": "str4d", 
    "type": "defect"
}
```

**Comment by str4d** [comment]
Trac update at 20170115T13:44:20: 
* zzz changed owner from "" to "str4d"
* zzz changed status from "new" to "assigned"

**Comment by str4d** [comment]
Trac update at 20170120T16:15:45: 
* str4d commented:

> The fix is in the repo, and will be released with 0.4.5 (which should be out soon).


* str4d changed keywords from "i2p-bote" to "I2P-Bote"
* str4d changed milestone from "undecided" to "0.9.29"
* str4d changed resolution from "" to "duplicate"
* str4d changed status from "assigned" to "closed"
